design/video-infrastructure.md

```diff
@@ -2,49 +2,49 @@
 
 ![](./assets/video-infrastructure.webp)
 
-When a streamer goes live they hit the first microservice called ingest.
+When a streamer goes live, they hit the first microservice called ingest.
 
 ## Ingest
 
-Ingest is responsible for handling the incoming go live requests. 
+Ingest is responsible for handling the incoming go-live requests. 
 
-It is agnostic to the protocol used to go live, for example we can support:
+It is agnostic to the protocol used to go live; for example, we can support the following:
 - [WebRTC](https://en.wikipedia.org/wiki/WebRTC)
 - [SRT](https://en.wikipedia.org/wiki/Secure_Reliable_Transport)
 - [RTMP](https://en.wikipedia.org/wiki/Real-Time_Messaging_Protocol)
 
-Once we get a new stream request we need to figure out who went live? What channel is the corrisponding stream request for (authentication), and what features the requested stream has access to.
+Once we get a new stream request, we need to figure out who went live. What channel is the corresponding stream request for (authentication), and what features does the requested stream have access to?
 
-We make a API call to the auth server to validate the incoming stream and then we make some API calls to create a stream target.
+We make an API call to the auth server to validate the incoming stream and then make some API calls to create a stream target.
 
-Once everything passes and we are all ready and set, we ingest the stream and convert it to a common protocol (SRT for example)
+Once everything passes, and we are all ready and set, we ingest the stream and convert it to a standard protocol (SRT, for example)
 
-At this point we can also pass the stream to other optional or future ideas (such as transcription or image detection).
+We can pass the stream to other optional or future ideas (such as transcription or image detection).
```

Comment:
"At this point in the pipeline we can insert other optional or future ideas for video processing" or something like that, not sure this is better but feels like it conveys the point more clear to me.

Reply:
yeah this makes sense